src/radical/pilot/agent/scheduler/continuous.py

```diff
@@ -226,6 +226,11 @@ def _find_resources(self, node, requested_cores, requested_gpus,
     # --------------------------------------------------------------------------
     #
     def _get_node_maps(self, cores, gpus, threads_per_proc):
+        '''
+        For each process (containing X threads) generate a process map that
+        requests 1 core per thread.  Core maps returns a list of lists of
+        process maps.
+        '''
```

Comment:
Jumana, I don't think this is very clear, yet.  I mean, it might be better than no description, but does not yet explain what the input parameters are, nor what kind of 'node_map' is meant in the method name...